### PR TITLE
pkg/trace/network: Prepare gadget for being added to local-gadget

### PR DESCRIPTION
**Pull request by blanquicet**
Prepare for https://github.com/inspektor-gadget/inspektor-gadget/issues/889

### CODE REVIEW

**Review by burak-ok — APPROVED** [review]
Looks good. Also thanks for simplifying the membervariables in `Event` + introducing a enum for the `RemoteKind`

**Review by mauriciovasquezbernal — APPROVED** [review]
LGTM.

File: pkg/gadgets/trace/network/types/types.go [hunk 1]
```diff
@@ -23,6 +23,18 @@ import (
...
+const (
+	RemoteKindPod     RemoteKind = "pod"
+	RemoteKindService RemoteKind = "svc"
+	RemoteKindOther   RemoteKind = "other"
+)
+
+func (r RemoteKind) String() string {
+	return string(r)
+}
+
```
[the remark this answers is not in the document]
Comment (blanquicet) [reply pkg/gadgets/trace/network/types/types.go]: Yep, it is not that useful.

**Comment by blanquicet** [comment]
Rebased to ensure the CI is all green after merging https://github.com/inspektor-gadget/inspektor-gadget/pull/1141.

**Comment by mauriciovasquezbernal** [comment]
What about removing the last commit of this PR and merging it? Then in a new PR you can add this to local-gadget once https://github.com/inspektor-gadget/inspektor-gadget/pull/1103 is merged.

**Pull request merged**